scripts/include/common.sh

```diff
@@ -36,10 +39,12 @@ set +x
 export DOCKER_USERNAME="${DOCKER_USERNAME:-}"
 export DOCKER_PASSWORD="${DOCKER_PASSWORD:-}"
 export CLUSTER_PASSWORD="${CLUSTER_PASSWORD:-password}"
-set -x
+
```

Comment:
I can put this back. I removed this because we assume output was on so after sourcing this file we always get the verbose output.